### PR TITLE
Prepare for 7.5.0

### PR DESCRIPTION
**Pull request by bperseghetti**
# 🎈 Release

Preparation for 7.5.0 release.

## Checklist
- [x] Asked team if this is a good time for a release
- [ ] There are no changes to be ported from the previous major version
- [ ] No PRs targeted at this major version are close to getting in
- [ ] Bumped minor for new features, patch for bug fixes
- [x] Updated changelog
- [x] Updated migration guide (as needed)
- [ ] Link to PR updating dependency versions in appropriate repository in [gazebo-release](https://github.com/gazebo-release) (as needed): <LINK>

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,5 +1,28 @@
...
+    * [Pull request #581](https://github.com/gazebosim/gz-math/pull/581)
+
+1. bazel: correctly export license
+    * [Pull request #586](https://github.com/gazebosim/gz-math/pull/586)
+
+1. Add missing eigen3.hh header for bazel build
+    * [Pull request #585](https://github.com/gazebosim/gz-math/pull/585)
+
+1. Revert changes break ABI
+    * [Pull request #Revert changes break ABI](https://github.com/gazebosim/gz-math/pull/Revert changes break ABI)
```
Comment (caguero) [inline Changelog.md:21]: Should we update the PR numbers here and in the next one?

Comment (bperseghetti) [reply Changelog.md:21]: Oh, I just used the script to summarize it. What PR# would we want to use a revert of the original PR number?

Comment (caguero) [reply Changelog.md]: I think you could remove this one as it restores the code as it used to be.

File: Changelog.md [hunk 2]
```diff
@@ -1,5 +1,28 @@
...
+
+1. Expose non-const reference to edges
+    * [Pull request #Expose non-const reference to edges](https://github.com/gazebosim/gz-math/pull/Expose non-const reference to edges)
```
Comment (caguero) [inline Changelog.md]: Maybe we didn't merge this one correctly. This is the commit. Maybe include the commit here? 
https://github.com/gazebosim/gz-math/commit/1649f76ddee22971c10c9c4bd51d16653145f8b9

File: Changelog.md (hunk 1 above)
Comment (bperseghetti) [reply Changelog.md]: @caguero fixed.

**Review by caguero — APPROVED** [review]

**Pull request merged**